server/config/incident.js

```diff
@@ -164,14 +164,13 @@ module.exports = {
       },
       {
         staffMemberNeedingMedicalAttention: {
-          responseType: 'requiredString',
-          validationMessage: 'What is the name of the staff who needed medical attention?',
+          sanitiser: (vals = []) => vals.reduce((res, val) => (val.name && val.name.trim() ? [...res, val] : res), []),
```

Comment:
Could this be clearer? I am not sure that I understand what it is doing.

Reply:
Good idea. I'll have a go.

Reply:
Have refactored and created a new function called inputsExcludingEmptyStrings()

Reply:
Reads better!